src/main/java/com/rakovets/course/java/core/practice/operators/Task03.java

```diff
@@ -37,6 +37,6 @@ public static void main(String[] args) {
     static float getTotalDepositAmount(int depositAmount, int depositYears, int depositAnnualPercentage) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return 0.0f;
+        return depositAmount+depositYears*((depositAmount * depositAnnualPercentage) / 100f);
```

Comment:
"Violation: There isn't separating white spaces for operators OR parameters
Solution: Add white spaces"